classes/Attachment.php

```diff
@@ -173,13 +173,24 @@ public static function deleteProductAttachments($idProduct)
      * @return bool true if success
      */
     public function attachProduct($idProduct)
+    {
+        return static::associateProductAttachment((int) $idProduct, (int) $this->id);
+    }
+
+    /**
+     * @param int $productId
+     * @param int $attachmentId
+     *
+     * @return bool true if success
+     */
+    public static function associateProductAttachment(int $productId, int $attachmentId): bool
```

Comment:
A `static` :scream:  ?

Reply:
yeah, I don't want to load full entity for association. At least if i make it public, IMO I should move this method to Product, because it makes much more sense to have Product->addAttachment, than Attachment->addProduct. I chose to leave this here as Product has too much methods already :smile:

Reply:
:ok_hand: